kano_world/session.py

```diff
@@ -76,11 +76,11 @@ def upload_profile_stats(self):
         # gender
         try:
             gender = profile['gender']
-            if gender == 'Boy':
+            if gender == _("Boy"):
                 gender = 'm'
-            elif gender == 'Girl':
+            elif gender == _("Girl"):
                 gender = 'f'
-            elif gender == "Wizard":
+            elif gender == _("Wizard"):
```

Comment:
Just wondering how this is going to impact our data tracking.. Is @delormev going to see the Spanish word for "Wizard"?